.rubocop.yml

```diff
@@ -1,6 +1,13 @@
+require:
+  - rubocop/cop/internal_affairs
+  - rubocop-graphql
+
 AllCops:
   TargetRubyVersion: 2.5
 
+GraphQL/ObjectDescription:
+  Enabled: false
```

Comment:
why should we disable this cop?

Reply:
https://github.com/DmitryTsepelev/rubocop-graphql/blob/a902c7187c3afc4a8f93b5f3ff6620ada6856d0a/.rubocop_todo.yml#L9-L14

Disabled in the above response.
`InternalAffairs/UndefinedConfig` considers it a violation when an option not defined in config/default.yml is referenced as cop_config[`Foo`] etc.

Therefore, we require `rubocop-graphql` at
https://github.com/DmitryTsepelev/rubocop-graphql/blob/47214f6350654c4af9f0fd6dcdabaeadc77eaadb/.rubocop.yml#L3

Then `GraphQL/ObjectDescription` will be enabled and the violation will be detected as follows
However, I don't think this is the cop that resolves the code under lib/rubocop/cop/graphql/*.

```
lib/rubocop/cop/graphql/argument_description.rb:21:13: C: GraphQL/ObjectDescription: Missing type description
      class ArgumentDescription < Base
            ^^^^^^^^^^^^^^^^^^^
lib/rubocop/cop/graphql/argument_name.rb:21:13: C: GraphQL/ObjectDescription: Missing type description
      class ArgumentName < Base
            ^^^^^^^^^^^^
lib/rubocop/cop/graphql/argument_uniqueness.rb:22:13: C: GraphQL/ObjectDescription: Missing type description
      class ArgumentUniqueness < Base
            ^^^^^^^^^^^^^^^^^^
lib/rubocop/cop/graphql/extract_input_type.rb:6:13: C: GraphQL/ObjectDescription: Missing type description
      class ExtractInputType < Base
            ^^^^^^^^^^^^^^^^
lib/rubocop/cop/graphql/extract_type.rb:6:13: C: GraphQL/ObjectDescription: Missing type description
      class ExtractType < Base
            ^^^^^^^^^^^
lib/rubocop/cop/graphql/field_definitions.rb:47:13: C: GraphQL/ObjectDescription: Missing type description
      class FieldDefinitions < Base # rubocop:disable Metrics/ClassLength
            ^^^^^^^^^^^^^^^^
lib/rubocop/cop/graphql/field_description.rb:21:13: C: GraphQL/ObjectDescription: Missing type description
      class FieldDescription < Base
            ^^^^^^^^^^^^^^^^
lib/rubocop/cop/graphql/field_hash_key.rb:26:13: C: GraphQL/ObjectDescription: Missing type description
      class FieldHashKey < Base
            ^^^^^^^^^^^^
lib/rubocop/cop/graphql/field_method.rb:26:13: C: GraphQL/ObjectDescription: Missing type description
      class FieldMethod < Base
            ^^^^^^^^^^^
lib/rubocop/cop/graphql/field_name.rb:25:13: C: GraphQL/ObjectDescription: Missing type description
      class FieldName < Base
            ^^^^^^^^^
lib/rubocop/cop/graphql/field_uniqueness.rb:29:13: C: GraphQL/ObjectDescription: Missing type description
      class FieldUniqueness < Base
            ^^^^^^^^^^^^^^^
lib/rubocop/cop/graphql/legacy_dsl.rb:22:13: C: GraphQL/ObjectDescription: Missing type description
      class LegacyDsl < Base
            ^^^^^^^^^
lib/rubocop/cop/graphql/object_description.rb:23:13: C: GraphQL/ObjectDescription: Missing type description
      class ObjectDescription < Base
            ^^^^^^^^^^^^^^^^^
lib/rubocop/cop/graphql/ordered_arguments.rb:51:13: C: GraphQL/ObjectDescription: Missing type description
      class OrderedArguments < Base
            ^^^^^^^^^^^^^^^^
lib/rubocop/cop/graphql/ordered_fields.rb:33:13: C: GraphQL/ObjectDescription: Missing type description
      class OrderedFields < Base
            ^^^^^^^^^^^^^
lib/rubocop/cop/graphql/resolver_method_length.rb:10:13: C: GraphQL/ObjectDescription: Missing type description
      class ResolverMethodLength < Base
            ^^^^^^^^^^^^^^^^^^^^
lib/rubocop/cop/graphql/unused_argument.rb:63:13: C: GraphQL/ObjectDescription: Missing type description
      class UnusedArgument < Base
            ^^^^^^^^^^^^^^
lib/rubocop/graphql/argument.rb:5:11: C: GraphQL/ObjectDescription: Missing type description
    class Argument
          ^^^^^^^^
lib/rubocop/graphql/argument/block.rb:5:11: C: GraphQL/ObjectDescription: Missing type description
    class Argument
          ^^^^^^^^
lib/rubocop/graphql/argument/block.rb:6:13: C: GraphQL/ObjectDescription: Missing type description
      class Block
            ^^^^^
lib/rubocop/graphql/argument/kwargs.rb:5:11: C: GraphQL/ObjectDescription: Missing type description
    class Argument
          ^^^^^^^^
lib/rubocop/graphql/argument/kwargs.rb:6:13: C: GraphQL/ObjectDescription: Missing type description
      class Kwargs
            ^^^^^^
lib/rubocop/graphql/class.rb:5:11: C: GraphQL/ObjectDescription: Missing type description
    class Class
          ^^^^^
lib/rubocop/graphql/field.rb:5:11: C: GraphQL/ObjectDescription: Missing type description
    class Field
          ^^^^^
lib/rubocop/graphql/field/block.rb:5:11: C: GraphQL/ObjectDescription: Missing type description
    class Field
          ^^^^^
lib/rubocop/graphql/field/block.rb:6:13: C: GraphQL/ObjectDescription: Missing type description
      class Block
            ^^^^^
lib/rubocop/graphql/field/kwargs.rb:5:11: C: GraphQL/ObjectDescription: Missing type description
    class Field
          ^^^^^
lib/rubocop/graphql/field/kwargs.rb:6:13: C: GraphQL/ObjectDescription: Missing type description
      class Kwargs
            ^^^^^^
lib/rubocop/graphql/schema_member.rb:5:11: C: GraphQL/ObjectDescription: Missing type description
    class SchemaMember
          ^^^^^^^^^^^^
lib/rubocop/graphql/swap_range.rb:5:12: C: GraphQL/ObjectDescription: Missing type description
    module SwapRange
           ^^^^^^^^^

66 files inspected, 30 offenses detected
```